app.js

```diff
@@ -29,6 +29,8 @@ var app = express();
 var routes = require('./routes');
 var routesUsers = require('./routes/users.js')
 
+const password_token = 'super-secret-f8ed84e8f41e4146403dd4a6bbcea5e418d24i7';
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in password_token.

<h6>
<a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 562</a> | Line 32</h6>